certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -0,0 +1,361 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Lead Loader",
+    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple API calls that can create or get the items needed to automate the Lead creation process.",
+    "contact": {
+      "name": "Andre Roberts",
+      "url": "https://phiquest.com/",
+      "email": "andre@phiquest.io"
```

Comment:
@andreroberts1119 would it be possible to use any email address for support purposes if you have one?

Reply:
Unfortunately, at the moment we do not have one setup. We are contacting another team to potentially set one up, and I will update the file if we have one to use.